### DISCUSSION TITLE
Fit aspect ratio to container height

### DISCUSSION THREAD

**Discussion by seleckis** [radix-ui/primitives#2834] [Help]
How can I make AspectRatio component fit parent element height? If I'm setting className for AspectRatio component it is being applied to child element not to external div with "data-radix-aspect-ratio-wrapper" attribute. I need AspectRatio element to fit height or width of parent element and be centered. Any suggestion?

**Comment by sebi75** [comment]
did you figure this out?

**Comment by seleckis** [comment]
Hi, @sebi75  That was long time ago, I don't remember, but try to make parent element as grid.
